components/classes/content-asset.schema.json

```diff
@@ -0,0 +1,149 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/content-asset",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Asset",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Content asset such as image or video.",
+  "definitions": {
+    "content-asset": {
+      "properties": {
+        "xdm:assetIdentityID": {
+          "title": "Asset Identity ID",
+          "type": "string",
+          "description": "Unique identifier of the asset perception."
+        },
+        "xdm:assetID": {
+          "title": "Asset ID",
+          "type": "string",
+          "description": "Unique identifier of the asset."
+        },
+        "xdm:assetName": {
+          "title": "Asset Name",
+          "type": "string",
+          "description": "Asset name."
+        },
+        "xdm:assetDescription": {
+          "title": "Asset Description",
+          "type": "string",
+          "description": "Asset description."
+        },
+        "xdm:assetType": {
+          "title": "Asset Type",
+          "type": "string",
+          "description": "Asset type such as image or video."
+        }
+      }
+    },
+    "content-asset-featurization": {
+      "properties": {
+        "xdm:contentFeaturization": {
+          "title": "Content Featurization",
+          "description": "Content featurization.",
+          "type": "object",
+          "properties": {
+            "xdm:orientation": {
+              "title": "Asset Orientation",
```

Comment:
Do you want it to be plain string, or an enum? Do you expect any other values besides `portrait`, `landscape` and `square`?

Reply:
This will be an enum. I will add the enum with: portrait, landscape, square.